faunadb-scala/src/main/scala/faunadb/values/Result.scala

```diff
@@ -1,6 +1,8 @@
 package faunadb.values
 
 object Result {
+  implicit def result2Iterable[A](r: Result[A]): Iterable[A] = if (r.isSuccess) List(r.get) else List.empty
```

Comment:
This has a side effect of ignoring failing results when flat mapping sequences of transformations. I think that behaviour can be misleading. A failure is something that we should not ignore silently in order to persevere the safety of our API.

Reply:
ins't that the purpose of `flatMap`? like:
```scala
    val x = Seq(Some(10), None)

    x.map(x => x) shouldBe Seq(Some(10), None)
    x.flatMap(x => x) shouldBe Seq(10)
```

that same occurs to `Result`
```scala
    val x = Seq(LongV(10).to[Long], StringV("20").to[Long])

    x.map(x => x) shouldBe Seq(VSuccess(10, FieldPathEmpty), VFail(List(FieldError("Expected Long value; found faunadb.values.StringV.", FieldPathEmpty))))
    x.flatMap(x => x) shouldBe Seq(10)
```